### ISSUE TITLE
Not able to reproduce results for CNN-for-Text-Classification

### ISSUE THREAD

**Issue by sksq96** [DSKSD/cs-224n-Pytorch#5]
Hey,

I am trying to reproduce [this notebook](https://nbviewer.jupyter.org/github/DSKSD/DeepNLP-models-Pytorch/blob/master/notebooks/08.CNN-for-Text-Classification.ipynb), but the loss do not go down as advertised. 

```
[0/5] mean_loss : 1.80
[1/5] mean_loss : 1.64
[2/5] mean_loss : 1.64
[3/5] mean_loss : 1.62
[4/5] mean_loss : 1.76
```

I am using PyTorch v0.4 on CUDA. My hypothesis is the newer version broke something.

```
>torch.__version__
'0.4.0a0+a3e9151'
```

Thanks!

**Comment by DSKSD** [comment]
Hi, 
Thanks for letting me know.
I don't test it on v0.4 yet. I'll check this out soon.
I wonder you initialize embedding matrix with pretrained Word2vec model.
If not, you should train your model much longer.
